### ISSUE TITLE
Geonode 2.8 and pycsw 2.0.3 OWSlib version missmatch

### ISSUE THREAD

**Issue by t-book** [GeoNode/geonode#3755]
Hi,

when installing requirements.txt from G28 I get following warning:

```
pycsw 2.0.3 has requirement OWSLib==0.10.3, but you'll have owslib 0.15.0 which is incompatible.
pycsw 2.0.3 has requirement pyproj==1.9.3, but you'll have pyproj 1.9.5.1 which is incompatible.
pycsw 2.0.3 has requirement Shapely==1.3.1, but you'll have shapely 1.5.17.post1 which is incompatible.
```
True, pycsw [2.0.3 pins OWSLib==0.10.3](https://github.com/geopython/pycsw/blob/2.0/requirements.txt#L3) where geonode 2.8 needs [owslib==0.15.0](https://github.com/GeoNode/geonode/blob/2.8.0/requirements.txt#L111)
Can this be ignored? As alternativ how about using pycsw 2.2 and OWSLib==0.16.0 ?

**Comment by afabiani** [comment]
Fixed on master / 2.7.x as we upgraded to pycsw 2.2.0

**Comment by t-book** [comment]
Thanks for the info Alessio!
